clients/admin-ui/cypress/fixtures/locations/list.json

```diff
@@ -112,44 +112,44 @@
   "regulations": [
     {
       "id": "gdpr",
-      "selected": false,
-      "name": "General Data Protection Regulation (GDPR)",
+      "selected": true,
+      "name": "GDPR (European Union)",
```

Comment:
the BE renamed these after the initial locations PR, so this is just making it batch what the BE would return now